diff --git a/packaging/build_rpm.sh b/packaging/build_rpm.sh
index cf781eb..7d2770f 100755
--- a/packaging/build_rpm.sh
+++ b/packaging/build_rpm.sh
@@ -40,7 +40,7 @@ files="CMakeLists.txt cmake cmake_uninstall.cmake.in cassconfig.hpp.in include s
 
 echo "Building version $version"
 
-libuv_version=$(rpm -q --queryformat "%{VERSION}" libuv)
+libuv_version=$(rpm -q --queryformat "%{VERSION}" libuv1)
 
 if [[ -e $libuv_version ]]; then
   echo "'libuv' required, but not installed"
diff --git a/packaging/cassandra-cpp-driver.spec b/packaging/cassandra-cpp-driver.spec
index bc42ff0..95163c3 100644
--- a/packaging/cassandra-cpp-driver.spec
+++ b/packaging/cassandra-cpp-driver.spec
@@ -14,9 +14,6 @@ Source1: cassandra.pc.in
 Source2: cassandra_static.pc.in
 
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root
-%if %{distnum} == 5
-BuildRequires: buildsys-macros >= 5
-%endif
 BuildRequires: cmake >= 2.6.4
 BuildRequires: libuv-devel >= %{libuv_version}
 BuildRequires: openssl-devel >= 0.9.8e
@@ -30,7 +27,7 @@ Language.
 Summary: Development libraries for ${name}
 Group: Development/Tools
 Requires: %{name} = %{epoch}:%{version}-%{release}
-Requires: libuv >= %{libuv_version}
+Requires: libuv1 >= %{libuv_version}
 Requires: openssl >= 0.9.8e
 Requires: pkgconfig
 
